software/earl/authenticator.go

```diff
@@ -32,6 +32,8 @@ const (
 	AuthExpired          = AuthResult(1)
 	AuthOkButOutsideTime = AuthResult(2) // User ok; time-of-day limit.
 	AuthOk               = AuthResult(42)
+	HolidayHiatusBegin   = 1482192000 // 2016-12-20
```

Comment:
Looks like the comment is off, and timezone-wise things don't line up with days here.